libs/permission-checker/src/Check/Ai/CanCreateAutomations.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker\Check\Ai;
+
+use Keboola\PermissionChecker\Exception\PermissionDeniedException;
+use Keboola\PermissionChecker\Feature;
+use Keboola\PermissionChecker\PermissionCheckInterface;
+use Keboola\PermissionChecker\Role;
+use Keboola\PermissionChecker\StorageApiToken;
+
+class CanCreateAutomations implements PermissionCheckInterface
+{
+    public function checkPermissions(StorageApiToken $token): void
+    {
+        if ($token->hasFeature(Feature::PROTECTED_DEFAULT_BRANCH)) {
+            throw new PermissionDeniedException(sprintf(
```

Comment:
V kontextu toho, že SOX neřešíme, tak jsem to tady na hulváta zakázal. Hypoteticky by šlo povolit zakládání pro reviewera a developera v branchi, ale to bych pak v AI service musel zjišťovat typ branche a zamotalo by se to a na to nemám náladu.